FirebaseAuthUI/FUIPasswordRecoveryViewController.xib

```diff
@@ -25,23 +26,35 @@
                     <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                     <color key="backgroundColor" red="0.93725490199999995" green="0.93725490199999995" blue="0.95686274510000002" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                     <view key="tableFooterView" contentMode="scaleToFill" id="hh3-Pu-5aV">
-                        <rect key="frame" x="0.0" y="592.5" width="375" height="100"/>
+                        <rect key="frame" x="0.0" y="896.5" width="375" height="100"/>
                         <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
                         <subviews>
-                            <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" bounces="NO" scrollEnabled="NO" editable="NO" text="Get instructions sent to this email that explain how to reset your password." textAlignment="natural" selectable="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ncz-Ir-7MQ">
-                                <rect key="frame" x="8" y="0.0" width="359" height="100"/>
-                                <color key="backgroundColor" red="0.0" green="0.0" blue="0.0" alpha="0.0" colorSpace="custom" customColorSpace="sRGB"/>
-                                <color key="textColor" red="0.0" green="0.0" blue="0.0" alpha="0.28999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
-                                <fontDescription key="fontDescription" type="system" pointSize="12"/>
-                                <textInputTraits key="textInputTraits" autocapitalizationType="sentences"/>
-                            </textView>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="Amh-KT-61X">
```

Comment:
Looks like this is causing Travis to fail. Since we still target iOS 8, stack view may not be available on all platforms.

Reply:
Migrated back to use plain auto layout.